app/util/api_version_handler.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+
+class APIVersionHandler(object):
+    # List of all versions of the API.
+    _versions = [
+        1,
+        2
+    ]
+
+    @classmethod
+    def resolve_version(cls, header: str, uri: str):
```

Comment:
Minor: wanna rename `header` to `accept_header` so that
1. It matches the name of the variable passed into this method
2. It's clearer which header this is